### PR TITLE
Update file

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,39 +25,34 @@ as stored procedures and server-side cursors.
...
 ```shell
 composer require yiisoft/db-pgsql
 ```
 
 ## Documentation
 
-English:
-
 - For config connection to PostgreSQL database check [Connecting PostgreSQL](https://github.com/yiisoft/db/blob/master/docs/en/connection/pgsql.md).
 - [Check the documentation](https://github.com/yiisoft/db/blob/master/docs/en/README.md) to learn about usage.
```
Comment (Tigrov) [inline README.md:43]: Could you check the links? They do not work.

File: README.md [hunk 2]
```diff
@@ -25,39 +25,34 @@ as stored procedures and server-side cursors.
 
 ## Support version
 
-|  PHP | Pgsql Version            |  CI-Actions
-|:----:|:------------------------:|:---:|
+| PHP | Pgsql Version | CI-Actions
+|----|------------------------|---|
 |**8.0 - 8.2**| **9.0 - 15.0**|[![build](https://github.com/yiisoft/db-pgsql/actions/workflows/build.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-pgsql/actions/workflows/build.yml) [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fdb-pgsql%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/db-pgsql/master) [![static analysis](https://github.com/yiisoft/db-pgsql/actions/workflows/static.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-pgsql/actions/workflows/static.yml) [![type-coverage](https://shepherd.dev/github/yiisoft/db-pgsql/coverage.svg)](https://shepherd.dev/github/yiisoft/db-pgsql)
```
Comment (Tigrov) [inline README.md:30]:
```suggestion
| PHP           | Pgsql Version  | CI-Actions
|---------------|----------------|-----------
| **8.0 - 8.3** | **9.0 - 15.0** |[![build](https://github.com/yiisoft/db-pgsql/actions/workflows/build.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-pgsql/actions/workflows/build.yml) [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fdb-pgsql%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/db-pgsql/master) [![static analysis](https://github.com/yiisoft/db-pgsql/actions/workflows/static.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-pgsql/actions/workflows/static.yml) [![type-coverage](https://shepherd.dev/github/yiisoft/db-pgsql/coverage.svg)](https://shepherd.dev/github/yiisoft/db-pgsql)
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="80px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
```
Comment (Tigrov) [inline README.md:3]:
```suggestion
        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px" alt="Yii">
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
     </a>
     <a href="https://www.postgresql.org/" target="_blank">
         <img src="https://www.postgresql.org/media/img/about/press/elephant.png" height="80px">
```
Comment (Tigrov) [inline README.md:6]:
```suggestion
        <img src="https://www.postgresql.org/media/img/about/press/elephant.png" height="80px" alt="PostgreSQL">
```

File: README.md (hunk 1 above)
Comment (luizcmarin) [reply README.md]: I'm checking all packages in alphabetical order (English). Now in db-mongoDB... But I already reviewed this for you...

**Review by Tigrov — APPROVED** [review]

**Review by vjik — APPROVED** [review]

**Pull request merged**